root/report/release_label_same_artist.tt

```diff
@@ -0,0 +1,15 @@
+[%- WRAPPER 'layout.tt' title=l('Releases with label the same as the artist') full_width=1 -%]
+
+<h1>[% l('Releases with label the same as the artist') %]</h1>
+
+<ul>
+    <li>[% l('This report lists releases that have their label set to be the same
+              as their artist.') %]</li>
```

Comment:
This is not exactly true - a label can't be the same as an artist. It's the names that are the same. So I would rephrase this as something like 'This report lists releases where the label name is the same as the artist name. Often this means the release is self-released, and the label {SpecialPurposeLabel|should be "[no label]" instead}.', {SpecialPurposeLabel: '/doc/Style/Unknown_and_untitled/Special_purpose_label'}

It's important that on seeing the report a user will understand why this report exists and what they are supposed to do with it :) Otherwise they'll think "ok, this report is here, but what's the point?".